tests/src/JIT/Regression/JitBlue/DevDiv_206786/handleMath.il

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Need to be careful about constant folding and handles
+// during prejitting.
```

Comment:
Just for my education, how are we ensuring that this test gets crossgen'ed before run?